src/network/TcpSocket.cpp

```diff
@@ -74,7 +74,7 @@ class SocketException : public Exception
 
 struct ConnectRequest
 {
-    TcpSocket * TcpSocket;
+    TcpSocket * Socket;
```

Comment:
it can be `TCPSocket` or `MySocket` or anything else, as long as it is not `TcpSocket`.